server/services/v1/database/base_runner.go

```diff
@@ -183,9 +184,13 @@ func (runner *BaseQueryRunner) mutateAndValidatePayload(coll *schema.DefaultColl
 		return doc, err
 	}
 
-	if err = coll.Validate(deserializedDoc); err != nil {
-		// schema validation failed
-		return doc, err
+	needValidation := api.GetHeader(ctx, api.HeaderSchemaSignOff) != "true"
```

Comment:
Can you move this in a helper method in `request.go`?

Reply:
By the way do we also send user-agent from client i.e. from which client request is issued?